acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -1,3 +1,4 @@
+import redirectLegacyBrowsers from '../../scripts/legacyBrowser.js';
```

Comment:
at a minimum, consider loading this dynamically much later at a point which could not hurt our page load performance.  Alternatively, let's look for a simpler solution or perhaps even retiring the redirect if we can get PM to approve.